recipes/openvino/meta.yaml

```
@@ -0,0 +1,521 @@
+  - url: https://github.com/opencv/ade/archive/58b2595a1a95cc807be8bf6222f266a9a1f393a9.tar.gz
+    sha256: db7759cc32107e713b9e503ddf71a54ba2c80de07daefcb1aa315507e706ce07
+    folder: openvino_sources/thirdparty/ade
+  - url: https://github.com/KhronosGroup/OpenCL-Headers/archive/1d3dc4e7562ac56ee8ab00607af7bd55fb091f22.tar.gz  # [win64 or linux64]
+    sha256: ae3815999e509a22f71141cea752572c5643abce9e840adec414cf50dfcfbcbd  # [win64 or linux64]
+    folder: openvino_sources/thirdparty/ocl/cl_headers  # [win64 or linux64]
+  - url: https://github.com/KhronosGroup/OpenCL-CLHPP/archive/89d843beba559f65e4a77833fcf8604e874a3371.tar.gz  # [win64 or linux64]
+    sha256: 9e60231bb89070058a49f39cda3c7945252feaff47a8cadd75548daaf377e914  # [win64 or linux64]
+    folder: openvino_sources/thirdparty/ocl/clhpp_headers  # [win64 or linux64]
+  - url: https://github.com/KhronosGroup/OpenCL-ICD-Loader/archive/9b5e3849b49a1448996c8b96ba086cd774d987db.tar.gz  # [win64 or linux64]
+    sha256: ffb1e97062a034275537de0be53cdf901f51535cba5de824a2334186d4f8dba7  # [win64 or linux64]
+    folder: openvino_sources/thirdparty/ocl/icd_loader  # [win64 or linux64]
+
+build:
+  number: 0
+  script_env:
+    - CI_BUILD_NUMBER={{ commit_number }}
+  run_exports:
+    - {{ pin_subpackage('libopenvino', max_pin='x.x.x') }}
+    - {{ pin_subpackage('libopenvino-intel-cpu-plugin', max_pin='x.x.x') }}  # [x86_64]
+    - {{ pin_subpackage('libopenvino-arm-cpu-plugin', max_pin='x.x.x') }}  # [aarch64 or arm64]
+    # - {{ pin_subpackage('libopenvino-intel-gpu-plugin', max_pin='x.x.x') }}
+    - {{ pin_subpackage('libopenvino-auto-plugin', max_pin='x.x.x') }}
+    - {{ pin_subpackage('libopenvino-hetero-plugin', max_pin='x.x.x') }}
+    - {{ pin_subpackage('libopenvino-auto-batch-plugin', max_pin='x.x.x') }}
+    - {{ pin_subpackage('libopenvino-ir-frontend', max_pin='x.x.x') }}
+    - {{ pin_subpackage('libopenvino-onnx-frontend', max_pin='x.x.x') }}
+    - {{ pin_subpackage('libopenvino-paddle-frontend', max_pin='x.x.x') }}
+    - {{ pin_subpackage('libopenvino-tensorflow-frontend', max_pin='x.x.x') }}
+  ignore_run_exports_from:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - pugixml # static package mistakenly adds run_exports
+    - tbb-devel # libopenvino and libopenvino already depend on tbb
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - git
+    - cmake >=3.13
+    - ninja
+    - gflags
+    - python >=3.7
+    - scons  # [aarch64 or arm64]
+    - libprotobuf
+    - sysroot_linux-64 =2.17  # [linux64]
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - numpy                                  # [build_platform != target_platform]
+    - libprotobuf                            # [build_platform != target_platform]
```

```suggestion
    - python >=3.7
    - scons  # [aarch64 or arm64]
    - libprotobuf
    - sysroot_linux-64 =2.17  # [linux64]
    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
    - numpy                                  # [build_platform != target_platform]
```
Remove redundant requirements.